android/app/src/main/java/com/safelymobileapp/MainApplication.java

```diff
@@ -6,6 +6,7 @@
 import com.facebook.react.ReactApplication;
 import com.oblador.vectoricons.VectorIconsPackage;
 import com.oblador.vectoricons.VectorIconsPackage;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
Why do you 3 times the same import ? Is this necessarry or I am missing on something ?

Reply:
Is this a generated file ?

Reply:
Thanks for noticing it, https://github.com/safely-app/Front-End-Mobile/pull/14/commits/70a0bf17e340d26abb15622750b38426a284ba68